test/lev-report/charts/CumulativeUsage.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { default as CumulativeUsage, maxDomain, ticks } from '../../../src/lev-report/charts/CumulativeUsage';
+import renderer from 'react-test-renderer';
+
+const data = [
+	{ month: '2016-11', count: 1930 },
+	{ month: '2016-12', count: 6084 },
+	{ month: '2017-01', count: 10134 },
+	{ month: '2017-02', count: 12605 },
+	{ month: '2017-03', count: 16326 },
+	{ month: '2017-04', count: 20228 },
+	{ month: '2017-05', count: 34890 },
+	{ month: '2017-06', count: 50379 },
+	{ month: '2017-07', count: 75494 },
+	{ month: '2017-08', count: 109769 },
+	{ month: '2017-09', count: 138787 },
+	{ month: '2017-10', count: 161880 },
+	{ month: '2017-11', count: 188782 },
+	{ month: '2017-12', count: 211174 },
+	{ month: '2018-01', count: 240620 },
+	{ month: '2018-02', count: 269532 },
+	{ month: '2018-03', count: 305283 },
+	{ month: '2018-04', count: 338861 },
+	{ month: '2018-05', count: 370632 },
+	{ month: '2018-06', count: 404105 },
+	{ month: '2018-07', count: 442241 },
+	{ month: '2018-08', count: 480672 },
+	{ month: '2018-09', count: 552781 },
+	{ month: '2018-10', count: 613444 },
+	{ month: '2018-11', count: 703487 },
+	{ month: '2018-12', count: 781353 },
+	{ month: '2019-01', count: 917086 },
+	{ month: '2019-02', count: 1036882 },
+	{ month: '2019-03', count: 1176872 },
+	{ month: '2019-04', count: 1310138 },
+	{ month: '2019-05', count: 1458269 },
+	{ month: '2019-06', count: 1617600 },
+	{ month: '2019-07', count: 1792798 },
+	{ month: '2019-08', count: 1963552 },
+	{ month: '2019-09', count: 2134995 },
+	{ month: '2019-10', count: 2334402 },
+	{ month: '2019-11', count: 2646704 },
+	{ month: '2019-12', count: 2997765 },
+	{ month: '2020-01', count: 3488802 },
+	{ month: '2020-02', count: 3918087 },
+	{ month: '2020-03', count: 4579347 },
+	{ month: '2020-04', count: 5149138 }
```

Comment:
This should be random or otherwise generated data.

Reply:
Well you say that, but if it were random the snapshot test would fail every time! What's wrong with the data as is?

Reply:
Commented on slack.

In terms of the snapshot, surely that could be regenerated?

Reply:
The data is now generated (consistently), so that should meet our needs. Fixed